app-typescript/components/avatar/avatar-wrapper.tsx

```diff
@@ -14,11 +14,14 @@ interface IPropsAvatarWrapper {
     };
     children: React.ReactNode;
     'data-test-id'?: string;
-    isEmpty?: boolean;
+    emptyLight?: boolean;
```

Comment:
That's not the best name. What about `theme: 'light' | 'dark'; // defaults to dark; only applies to placeholder image`

Reply:
In this case, I would go with something that is neutral regarding the overall theme. For example, `appearance: 'subtle' | 'strong';` or something similar.

Reply:
That's even worse than `emptyLight` :) what about `noAvatarPlaceholderColor: 'light' | 'dark';`. Or maybe even better - make it gray so we don't need to configurable at all :upside_down_face:

Reply:
How come? It's logical. I want to avoid 'light' and 'dark' because if you change the SD theme, it's no longer accurate. The 'light' option would actually be darker than the 'dark' in the dark SD theme 🙂. Hence, 'subtle' and 'strong', it holds true in both themes. As for the prop name, I don't have any preferences.

Reply:
> The 'light' option would actually be darker than the 'dark' in the dark SD theme

Makes sense. So what it means in the end is whether theme color is used or inverse. What about
`noAvatarPlaceholderColor: 'theme' | 'theme-inverse'`?